### ISSUE TITLE
Ghost Ice villages

### ISSUE THREAD

**Issue by GundunUkan** [AlexModGuy/Ice-and-Fire-Suggestions#345]
This is to go with the removal of the Ice Villagers. When they are removed, leave the villages to indicate there where once ice villagers but they are now brought to extinction by the ice dragons. Maybe make them generate with a bit of destruction caused by ice dragons, like a few craters and dragon ice here and there.

**Comment by TheOneTraveler** [comment]
i agree with this, but also make uh, ghosts spawn there in abundance once ghosts are added. could serve as a form of dungeon, you could leave chests thoughout the remaining houses with special loot which could include their robes and the fishing spear!
